accounting/src/creditcommons/routes.ts

```diff
@@ -19,6 +38,16 @@ import {
 export function getRoutes(controller: SharedController) {
   const router = Router()
 
+  /**
+   * Configure the trunkward CC node. Requires admin.
+   */
+  router.post('/:code/creditCommonsNodes', userAuth(Scope.Accounting), checkExact(CreditCommonsValidators.isGraft()),
```

Comment:
shouldn't it be `/:code/cc/nodes`?

Reply:
Ah yes, I have no particular preference! This was just following the pattern of using the database table name.